pixi.toml

```diff
@@ -586,18 +589,18 @@ platforms = ["linux-64", "linux-aarch64", "osx-arm64", "osx-64", "win-64"]
 [feature.python-pypi.dependencies]
 # Note these are the same as in `python-dev` but we need to repeat them here because the two are mutually exclusive.
 opencv = ">4.6"
-numpy = "<2"    # Rerun still needs numpy <2. Enforce this outside of the pypi dep tree so we pick up the conda version.
+numpy = ">=2"   # Rerun still needs numpy <2. Enforce this outside of the pypi dep tree so we pick up the conda version.
 
 
 [feature.python-pypi.pypi-dependencies]
-rerun-sdk = "==0.17.0"
-rerun-notebook = "==0.17.0"
+rerun-sdk = "==0.22.1"
+rerun-notebook = "==0.22.1"
 
 # EXAMPLES ENVIRONMENT
 [feature.examples-common]
 # Ideally we would could remove`linux-aarch64` from here, but it breaks other things.
 # See: https://github.com/prefix-dev/pixi/issues/1051
-platforms = ["linux-64", "linux-aarch64", "osx-arm64", "osx-64", "win-64"]
+platforms = ["linux-64", "osx-arm64", "win-64"]
```

Comment:
While `osx-64` is in definitely in the past, im afraid `linux-aarch64` is in the future. We'll have to deal with that sooner than later.